features/Steps/UtilitySteps.php

```diff
@@ -24,6 +24,14 @@ public function iCreateARandomNameOfCharacters(int $nrOfCharacters): void
         $this->variableState->setRandomVariable('name', $nrOfCharacters);
     }
 
+    /**
+     * @Given I set the value of name to value :value
```

Comment:
No need for the last "value" in the sentence, it can just be "I set the value of name to"